docs/manual/setup/getting-started.md

```diff
@@ -0,0 +1,181 @@
+
+Once we have Vector installed, let's check to make sure that it's working correctly:
+
+```shell
+vector --version
+```
+
+## Configure Vector
+
+Vector topologies are defined with a [configuration file][docs.setup.configuration] that tells it what
+[components][pages.components] to run and how they should interact. Vector topologies are made up of three
+types of components:
+
+* [**Sources**][docs.sources] collect or receive data from observability data sources into Vector
+* [**Transforms**][docs.transforms] can manipulate or change that observability data inside Vector
+* [**Sinks**][docs.sinks] send data onwards from Vector to external services or destinations
+
+Let's create a configuration file:
+
+```toml title="vector.toml"
+[sources.in]
+  type = "stdin"
+
+[sinks.out]
+  inputs = ["in"]
+  type = "console"
+  encoding.codec = "text"
+```
+
+Each component has a unique identifier and is prefixed with the type of component, for example, `sources` for a
+source. Our first component, `sources.in`, uses the [`stdin` source][docs.sources.stdin] which tells Vector to receive
+data over stdin and is identified as `in`.
+
+Our second component, `sinks.out`, uses the [`console` source][docs.sinks.console] which tells Vector to simply print
+the data to stdout. The `encoding.codec` option tells Vector to print the data as plain text (unencoded).
+The `inputs` option creates a pipeline from our source, which receives events, to our sink, which sends those
+events onwards. The `inputs` option in our `sinks.out` component tells Vector where this sink's events are coming
+from. In our case, events are received from our other component, the source identified as `in`.
+
+## Hello World!
+
+That's it for our first config. Now let's pipe an event through it:
+
+```bash
+echo 'Hello World!' | vector --config ./vector.toml
+```
+
+The `echo` statement sends a single log to Vector via stdin. The `vector...` command starts Vector with our
+previously created config file.
+
+The event we've just sent will be received by our `sources.in` component, then sent onto the `sinks.out` component,
+which will, in turn, echo it back to the console:
+
+```text
+... some logs ...
+Hello World!
+```
+
+> Hey, if you want to see something cool, try setting `encoding.codec = "json"` in the sink config.
+
+## Hello World Mark II
+
+The echoing of events isn't very exciting. Let's see what we can do with some real observability data.
+Let's take a classic problem, collecting and processing Syslog events, and see how Vector handles it.
+
+To do this, we're going to add two new components to our configuration file. Let's look at our updated
+configuration now:
+
+```toml title="vector.toml"
+[sources.generate_syslog]
+  type = "generator"
+  format = "syslog"
+  count = 100
+
+[transforms.remap_syslog]
+  inputs = [ "generate_syslog"]
+  type = "remap"
+  source = '''
+    . = parse_syslog!(.message)
+  '''
```

Comment:
```suggestion
  source = '''
  structured = parse_syslog!(.message)
  . = merge(., structured)
  '''
```

Reply:
This feels like added complexity for a getting started guide? Can you expand on the why of the change?

Reply:
See #6171. Overwriting the root object is generally not desired.

Reply:
And I agree! Hence why I opened #6171.